### ISSUE TITLE
kdeApplications.{calendarsupport,kaddressbook} broken on master

### ISSUE THREAD

**Issue by JJJollyjim** [NixOS/nixpkgs#107503]
**Describe the bug**
These packages fail to build with:
```
/build/kaddressbook-20.08.2/src/printing/mike/mikesstyle.cpp: In member function 'virtual void KABPrinting::MikesStyle::print(const List&, KABPrinting::PrintProgress*)':
/build/kaddressbook-20.08.2/src/printing/mike/mikesstyle.cpp:126:66: error: no matching function for call to 'QPrinter::setPageMargins(int, int, int, int, QPrinter::Unit)'
  126 |     printer->setPageMargins(20, 20, 20, 20, QPrinter::DevicePixel);
      |                                                                  ^
```

https://hydra.nixos.org/build/133580315/nixlog/2
https://hydra.nixos.org/build/133595957/nixlog/2

In turn this breaks everything akonadi-based (kmail, korganizer...)

@ttuegel @vandenoever @nyanloutre

**Comment by ttuegel** [comment]
This should be fixed by the KDE Applications 20.12.0 update.

**Comment by ttuegel** [comment]
Fixed in #107662.
